satpy/etc/readers/aws_l1b_nc.yaml

```diff
@@ -0,0 +1,535 @@
+reader:
+  name: aws_l1b_nc
```

Comment:
This should be `mws_l1b_nc` based on our naming convention using the sensor name and what OSCAR tells us it is.